discovery-infra/test_infra/consts.py

```diff
@@ -13,6 +13,7 @@
 VALIDATION_TIMEOUT = 6 * 60
 NTP_VALIDATION_TIMEOUT = 10 * 60
 OCS_VALIDATION_TIMEOUT = 10 * 60
+CNV_VALIDATION_TIMEOUT = 10 * 60
```

Comment:
Where is this used?

Reply:
I want to use it as a timeout to wait on cnv related validations. Used in downstream repo.